nodes/clusters/nodes-cluster-cgroups-2.adoc

```diff
@@ -1,18 +1,15 @@
 :_content-type: ASSEMBLY
 :context: nodes-cluster-cgroups-2
 [id="nodes-cluster-cgroups-2"]
-= Enabling Linux control group version 2 (cgroup v2)
+= Configuring the Linux cgroup version on your nodes
 include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control group version 2] (cgroup v2) in your cluster by editing the `node.config` object. Enabling cgroup v2 in {product-title} disables all cgroups version 1 controllers and hierarchies in your cluster. cgroup v1 is enabled by default.
+By default, {product-title} uses  link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v1.html[Linux control group version 1] (cgroup v1) in your cluster. You can switch to link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control group version 2] (cgroup v2), if needed, by editing the `node.config` object. Enabling cgroup v2 in {product-title} disables all cgroup version 1 controllers and hierarchies in your cluster. 
 
 cgroup v2 is the next version of the Linux cgroup API. cgroup v2 offers several improvements over cgroup v1, including a unified hierarchy, safer sub-tree delegation, new features such as link:https://www.kernel.org/doc/html/latest/accounting/psi.html[Pressure Stall Information], and enhanced resource management and isolation. 
 
-:FeatureName: {product-title} cgroups version 2 support
```

Comment:
There's a remaining link to the FeatureGates section down in the "Additional resources" below in this assembly.